README.md

```diff
@@ -24,9 +24,31 @@ python -m venv venv
 pip install -r requirements.txt
 ```
 
+## Startup
+The app runs on a `gunicorn` server which is set up via the `start.sh` script. This also sets up the database. 
+
 ## Database
 The database schema can be recreated at any time via `bot/app/database/db_create.py`.
-Database population is twofold: users are automatically added via the `\start` command upon first conversation initilization. Events, artists and their relations are added via `db_update.py` also residing in the `app/database` subdirectory. It provides a very small CLI to insert new data, using the classes provided in `db_objects.py`.
+Database population is twofold: users are automatically added via the `\start` command upon first conversation initilization. Events, artists and their relations are added via `db_update.py` also residing in the `app/database` subdirectory. It provides a very small CLI to insert new data, using the classes provided in `db_objects.py`. Manual inputs to the db have to look like this:
+
+- __Events__:
+    - eName: string
+    - date: string, Format DD.MM.YYYY
+    - time: string, Format HH:MM
+    - description: string, must not exceed 150 characters
+    - admission: string
+    - ePicID: string, 6 ciphers long, begins with a 0. followed by file ending - e.g. 002378.jpg
+- __Artist__:
+    - aName: string
+    - website: string
+    - soundcloud: string
+    - bandcamp: string
+    - bio: string, must not exceed 150 characters
+    - aPicID: string, 6 ciphers long, begins with a 1. followed by file ending - e.g. 100356.jpg
+- __PlaysAt__:
+    - aName: str, same as Artist.aName
+    - eName: str, same as Event.eName
+    - date: str, same as Event.date
```

Comment:
I would not post this here, but link to each class in the source code. Because if any of this changes, you will need to change it in two places now. In the future we can consider hosting the docs via Sphinx or something like that, this will automatically render docstring comments in the source code.

Reply:
will fix this after we have the first version deployed on the server and then write down the readme properly